gulpfile.ts

```diff
@@ -159,6 +160,22 @@ function writeFilePromise(path, content) {
   });
 }
 
+function sitemap(cards: SingleCardJSON[]): SingleCardJSON[] {
+  const baseurl = 'http://www.aliemcards.com';
+  let sitemap = `${baseurl}\n`;
```

Comment:
This is fine, but my preference would be to use a single, functional, immutable string, like so:

```js
const str = `
${baseurl}
${baseurl}/about
${baseurl}/categories
${baseurl}/cards
${cards.map(card => `${baseurl}/cards/${card.id}`).join('\n')}
`;
```

Your call